src/database/queries/baseFieldsCopyTasks/selectById.sql

```diff
@@ -0,0 +1,3 @@
+SELECT base_fields_copy_task_to_json(base_fields_copy_tasks.*) as "object"
```

Comment:
There is a [known limitation](https://docs.sqlfluff.com/en/stable/production/github_actions.html) of 10 annotations from the SQLFluff check shown in the diff UI, but there are more that you can see in the check output or by running locally.

```suggestion
SELECT base_fields_copy_task_to_json(base_fields_copy_tasks.*) AS object
```